<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Error_Message</class>
 <widget class="QDialog" name="Error_Message">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snap!!</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pics/Logo.png</normaloff>pics/Logo.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.800000000000000</double>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="Oh_No">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Pics/Oh_No.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="Oh_No_Message">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Pics/Oh_No.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="OKEr_PB">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
